src/SurveyService.Models.Dto/Requests/Option/CreateOptionRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto;
+
+public record CreateOptionRequest
+{
+  public Guid QuestionId { get; set; }
+  [Required]
+  public string Content { get; set; }
+  public bool IsCustom { get; set; }
```

Comment:
I think that shouldn't be in the request: we should not allow anyone from the outside to set or get this value.
If option came from its controller: we should set it in the DD model to custom.
If option was created simultaneously with question we should set it to non-custom

Reply:
Marina and I decided that it would be better to leave it, so that in case of changes in the technical task, it would be possible to get IsCustom from the frontend.